src/codeu/chat/common/VersionInfo.java

```diff
@@ -0,0 +1,24 @@
+
+
+package codeu.chat.common;
+import codeu.chat.util.Uuid; 
+
+public final class VersionInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  public Uuid temp = null;
+  
+  public VersionInfo() {
+  	try {
+  		temp = Uuid.parse(SERVER_VERSION);
+    }
+    catch (Exception ex) {
+    	System.out.println("ERROR");
+    }
+    this.version = temp;
+  }
+  public VersionInfo(Uuid version) {
```

Comment:
since you have two constructors, please elaborate where/when each one is used (client/server)